src/react/deckgl.js

```diff
@@ -20,8 +20,8 @@
 
 import React, {createElement, cloneElement} from 'react';
 import autobind from './utils/autobind';
-import {experimental, log} from '../core';
-const {DeckGLJS} = experimental;
+import {experimental} from '../core';
+const {DeckGLJS, log} = experimental;
```

Comment:
Why is log in experimental, but this change does match reset of the code.

Reply:
> Why is log in experimental

We don't want deck.gl to export a frozen logging API. It is something we use internally and want to be able to change between releases.

Perhaps we should distinguish between exports that are `internal` (for other submodules, such as deck.gl/react) and `experimental` (for adventurous users).